prime-dotnet-webapi/Views/Emails/SiteApprovedPharmaNetAdministratorEmail.cshtml

```diff
@@ -0,0 +1,31 @@
+@using Prime.Services
+@model EmailParams
+
+@{
+  var site = Model.Site;
+}
+
+<p>
+  @site.DoingBusinessAs has been approved for PharmaNet access by the Ministry
+  of Health. The site’s Signing Authority provided your name and contact information
+  as the PharmaNet Administrator for this site. Your PharmaNet software vendor may
+  contact you to complete the site setup process, and to help set up user access accounts.
+</p>
+
+<p>
+  Individuals who will be accessing PharmaNet at your site should enrol in PRIME
+  now if they have not already done so. For more information, please visit
+  http://www.gov.bc.ca/pharmanet/PRIME. [for private community practice only:

```

Comment:
I think we should make the url's into links?

Reply:
> I think we should make the url's into links?

Yep, fixed.